dali/kernels/imgproc/resample/resampling_setup.cc

```diff
@@ -58,12 +58,19 @@ void SeparableResamplingSetup<spatial_ndim>::SetFilters(
 
     auto fdesc = desc.out_shape()[axis] < in_size ? params[dim].min_filter
                                                   : params[dim].mag_filter;
+
+    if (fdesc.antialias && fdesc.type == ResamplingFilterType::Linear)
+      fdesc.type = ResamplingFilterType::Triangular;
+    else if (!fdesc.antialias && fdesc.type == ResamplingFilterType::Triangular)
+      fdesc.type = ResamplingFilterType::Linear;
+
```

Comment:
Detected when running a debug build of dali_kernel_test - triggered an assert in line 30.